### PR TITLE
Use 'HEAD' in VERSION file on master branch.

### PR DESCRIPTION
**Pull request by StevenYCChou**
The build information of the binary file will be accurate when anyone
builds from master.

Also guide user to release page on GitHub as source of truth.

### CODE REVIEW

**Comment by StevenYCChou** [comment]
This PR will resolve #128.

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.4.0
+latest
```
Comment (jkohen) [inline VERSION]: Should this be HEAD?

Comment (StevenYCChou) [reply VERSION]: DONE.

**Comment by StevenYCChou** [comment]
@jkohen Please take a look on the latest change. VERSION is not mentioned in the Makefile, so I added the documentation in the release script on why we update VERSION file.

File: release.sh [hunk 2]
```diff
@@ -19,6 +19,8 @@ fi
 ################################
 
 # 1. Update file `VERSION` with the numeric version, e.g. `0.3.1`.
+#    Note that VERSION file is required for building tool promu:
```
Comment (jkohen) [inline release.sh]: Nit, remove "Note that", it doesn't add any value.
Write "is required by" instead of "for".

Comment (StevenYCChou) [reply release.sh]: Thanks for the wording suggestion to make it precise. Updated.

**Pull request merged**